changelogs/fragments/82678-role-entrypoint-attributes.yml

```
@@ -0,0 +1,2 @@
+deprecated_features:
+  - "ansible-doc - role entrypoint attributes are deprecated and will no longer be shown in ansible-doc from ansible-core 2.20 on (https://github.com/ansible/ansible/issues/82639, https://github.com/ansible/ansible/pull/82678)."
```

'and eventually' .. right now this PR restores them doesn't it?

---

Do you mean like
```suggestion
  - "ansible-doc - role entrypoint attributes are deprecated and eventually will no longer be shown in ansible-doc from ansible-core 2.20 on (https://github.com/ansible/ansible/issues/82639, https://github.com/ansible/ansible/pull/82678)."
```
?

---

even though you specified the version, previous phrasing could confuse people to think it was immediate, thanks!